packages/commuter-cli/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@nteract/commuter-cli",
-  "version": "0.1.2",
+  "version": "0.1.4",
   "description": "cli to setup commuter client and server",
   "bin": {
     "commuter": "lib/index.js"
   },
   "scripts": {
     "start": "concurrently \"npm run server\" \"npm run client\"",
-    "start:production": "npm run server",
+    "start:production": "NODE_ENV=production&&npm run server",
```

Comment:
no space is on purpose. Its to avoid trailing spaces as per [this](http://stackoverflow.com/questions/25112510/how-to-set-environment-variables-from-within-package-json-node-js)